osp/migrations/0018_remove_zulipstat_last_activity.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.0.7 on 2020-08-15 00:24
```

Comment:
Please optimize the migrations, I guess there are a few which just add/remove same field